src/account/SignUp.jsx

```diff
@@ -29,7 +28,7 @@ export default function SignUpScreen(props) {
     }) */
     try {
       await createUserWithEmailAndPassword(auth, value.email, value.password);
-      props.SetSignUp(true);
+      alert("Registered!");
```

Comment:
Do you mean "Sign Up Successfully"? Please keep the wording consistent.

Reply:
changed it